tests/compile/cpp_temp_assignment.pyx

```diff
@@ -3,34 +3,72 @@
 
 cdef extern from *:
     """
-    #if __cplusplus >= 201103L
+    class NoAssignIterator {
+        public:
+        #if __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER >= 1600)
```

Comment:
If you add `#tag: cpp11` at the top of the file then it'll only run with a suitable compiler and you can remove these `#if` bits.

(I think I should have done this when I originally made this file...)

Reply:
Thanks for the pointer. I had to use cpp,cpp11 -- otherwise runtests appear to try to run the test under compiler=c --std=cpp11.